spec/models/miq_server/update_management_spec.rb

```diff
@@ -202,6 +203,10 @@
   end
 
   context "#apply_updates" do
+    before do
+      MiqDatabase.stub(:postgres_package_name => "postgresql-server")
```

Comment:
These stubs seems better than setting whatever the ENV key is that gets populated on appliances.